### DISCUSSION TITLE
Adding page switch like on Fumadocs

### DISCUSSION THREAD

**Discussion by antoineross** [fuma-nama/fumadocs#603] [Ideas]
Hi! I am trying to use fumadocs for an open source project and I'd like to be able to use the toggle like in Fumadocs. Is that natively supported or do we need to make our own custom code for it? 

<img width="311" alt="Screenshot 2024-07-15 at 10 40 02 PM" src="https://github.com/user-attachments/assets/b9ce4e30-9479-458e-ad7d-e64c7f4b2d91">

**Comment by fuma-nama** [comment]
You can search for Root Toggle.

**Reply by antoineross** [reply]
Oh sorry...I didn't check the docs properly thank you so much I appreciate it
